dataland-email-service/src/main/kotlin/org/dataland/datalandemailservice/services/templateemail/TemplateEmailFactory.kt

```diff
@@ -34,7 +34,7 @@ abstract class TemplateEmailFactory(
      * @param receiverEmail the single receiver email address
      * @param properties the properties required to fill the HTML template
      */
-    fun buildEmail(
+    open fun buildEmail(
```

Comment:
this is just cosmetics, right? or does it do something?

Reply:
This is necessary so that the subclasses of the factory can override the build method and retrieve the subscription UUID for the recipient.